test/suites/integration/bsp/storage-delete.test.ts

```diff
@@ -55,7 +55,12 @@ describeBspNet(
       await userApi.assert.eventPresent("fileSystem", "StorageRequestRevoked");
 
       // Unpause bsp three
-      await userApi.docker.resumeBspContainer({ containerName: "sh-bsp-three" });
+      await userApi.docker.resumeBspContainer({
+        containerName: "sh-bsp-three"
+      });
+
+      // TODO: create an RPC to automatically execute everything below
+      // TODO: everything below should be removed and replaced with other testing logic
```

Comment:
I disagree because then it is up to the user to manually assemble all the data like proofs and such to call an extrinsic, which I might add has to be waited for a certain amount of time before you can call it successfully. that's quite a lot of burden on a user